test/e2e-go/features/participation/participationExpiration_test.go

```diff
@@ -76,7 +76,7 @@ func testExpirationAccounts(t *testing.T, fixture *fixtures.RestClientFixture, f
 		_, currentRound := fixture.GetBalanceAndRound(richAccount)
 		// account adds part key
 		partKeyFirstValid := uint64(0)
-		partKeyValidityPeriod := uint64(150)
+		partKeyValidityPeriod := uint64(150) // TODO: maybe increase?
```

Comment:
What are the units?  I would have guessed rounds, but that would be very short.

Reply:
Yes they are rounds and it is very short, maybe we should put 1000-3000 there instead, as no State Proof keys will be generated for such a short interval.